puppet/modules/jenkins_node/manifests/init.pp

```diff
@@ -137,4 +137,22 @@
   file { '/etc/cron.daily/jenkins_cleaner':
     ensure => absent,
   }
+
+  file { '/usr/local/sbin/reboot-jenkins-node':
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+    source => 'puppet:///modules/jenkins_node/reboot-jenkins-node',
```

Comment:
I tend to inline the content to avoid the fileserver. For small scripts this is faster and in theory could could use a cached catalog to reapply, even if we really don't use this. The biggest win is really that it will fail on catalog compilation if the file doesn't exist so your unit tests can capture it.
```suggestion
    content => file("${module_name}/reboot-jenkins-node"),
```